server/plugin/mm_34646_token_refresh.go

```diff
@@ -20,9 +19,10 @@ func (p *Plugin) forceResetAllMM34646() error {
 	ctx := context.Background()
 
 	time.Sleep(delayToStart)
-	data, appErr := p.API.KVGet(mm34646DoneKey)
-	if appErr != nil {
-		return errors.Wrap(appErr, "failed check whether MM-34646 refresh is already done")
+	var data []byte
+	err := p.client.KV.Get(mm34646DoneKey, &data)
```

Comment:
Do you think we can rename this var to something else? It doesn't provide much context reading the code

Reply:
To which variable are you referring?